src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
@mjwitte and @Myoldmopar: The change I made yesterday removed the _DaylCtrl from the Daylighting Reference Point Illuminance variables which is the right name for the variables. That does not produce any warning messages and when I manually added the 1, 2, 3 that is when an warning message is generated. So the reason I thought everything working yesterday is because the files were not generating warnings, which was my goal.

On the other hand, looking at the CSV generated, those three variables produce all zeros. A different issue that I will try to resolve.